go.mod

```diff
@@ -17,19 +18,46 @@ require (
 	github.com/onsi/gomega v1.10.0 // indirect
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/viper v1.5.0
+	github.com/sirupsen/logrus v1.5.0
+	github.com/spf13/viper v1.6.1
 	golang.org/x/net v0.0.0-20200114155413-6afb5195e5aa
+	golang.org/x/sync v0.0.0-20200317015054-43a5402ce75a
 	google.golang.org/grpc v1.27.1
-	gopkg.in/yaml.v2 v2.2.4
+	gopkg.in/yaml.v2 v2.2.8
 )
 
 replace (
 	github.com/census-instrumentation/opencensus-proto v0.1.0-0.20181214143942-ba49f56771b8 => github.com/census-instrumentation/opencensus-proto v0.0.3-0.20181214143942-ba49f56771b8
 	github.com/networkservicemesh/networkservicemesh/controlplane => github.com/tiswanso/networkservicemesh/controlplane v0.2.0-vl3
 	github.com/networkservicemesh/networkservicemesh/controlplane/api => github.com/tiswanso/networkservicemesh/controlplane/api v0.2.0-vl3
 	github.com/networkservicemesh/networkservicemesh/forwarder/api => github.com/tiswanso/networkservicemesh/forwarder/api v0.2.0-vl3
+	github.com/networkservicemesh/networkservicemesh/k8s => github.com/networkservicemesh/networkservicemesh/k8s v0.0.0-20200525171636-c1b195957964
+	github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis => github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis v0.0.0-20200525171636-c1b195957964
```

Comment:
These need to be `v0.2.0-vl3` 
```
github.com/tiswanso/networkservicemesh/k8s v0.2.0-vl3
github.com/tiswanso/networkservicemesh/k8s/pkg/apis v0.2.0-vl3
```

The repo has these tags:
```
k8s/pkg/apis/v0.2.0-vl3
k8s/v0.2.0-vl3
```

Reply:
done